hack/ocm-minikube-ramen.sh

```diff
@@ -0,0 +1,360 @@
+	rook_ceph_branch_checkout_undo
+}
+exit_stack_push unset -f rook_ceph_deploy
+rook_ceph_undeploy()
+{
+	rook_ceph_branch_checkout ${1}/..
+	PROFILE=${3} ${1}/minikube-rook-setup.sh delete
+	PROFILE=${2} ${1}/minikube-rook-setup.sh delete
+	rook_ceph_branch_checkout_undo
+}
+exit_stack_push unset -f rook_ceph_undeploy
+ramen_branch_name=pr47_pr58_rbac2360357
+exit_stack_push unset -v ramen_branch_name
+ramen_branch_build()
+{
+	set -- ${1} ${ramen_branch_name}
+	git --git-dir ${1}/.git --work-tree ${1} checkout main -b ${2}
+	exit_stack_push git --git-dir ${1}/.git --work-tree ${1} checkout -
+	git --git-dir ${1}/.git --work-tree ${1} pull --rebase https://github.com/ramendr/ramen main
+	#git --git-dir ${1}/.git --work-tree ${1} pull --rebase https://github.com/BenamarMk/ramen avr_per_subscription_placement       #51
+	#git --git-dir ${1}/.git --work-tree ${1} pull --rebase https://github.com/ShyamsundarR/ramen pr-51-1
+	#git --git-dir ${1}/.git --work-tree ${1} pull --rebase https://github.com/BenamarMk/ramen add_avr_plrule_to_manage_user_plrule #55
+	git --git-dir ${1}/.git --work-tree ${1} pull --rebase https://github.com/ShyamsundarR/ramen 2360357b37e76f1bcd5909598d49c1756780cd8e
+	git --git-dir ${1}/.git --work-tree ${1} pull --rebase https://github.com/BenamarMk/ramen add_preferredcluster_targetcluster    #58
+	# controllers/applicationvolumereplication_controller.go
+	# config/crd/bases/ramendr.openshift.io_applicationvolumereplications.yaml
+	git --git-dir ${1}/.git --work-tree ${1} push git@github.com:hatfieldbrian/ramen ${2}
+	exit_stack_pop
+}
+exit_stack_push unset -f ramen_branch_build
+ramen_branch_checkout()
+{
+	set -- ${1} ${ramen_branch_name}
+	git --git-dir ${1}/.git --work-tree ${1} checkout main
+	exit_stack_push git --git-dir ${1}/.git --work-tree ${1} checkout -
+	git --git-dir ${1}/.git fetch https://github.com/hatfieldbrian/ramen ${2}:${2}
+	exit_stack_pop
+	git --git-dir ${1}/.git --work-tree ${1} checkout ${2}
+	exit_stack_push git --git-dir ${1}/.git --work-tree ${1} checkout -
+}
+exit_stack_push unset -f ramen_branch_checkout
+ramen_branch_checkout_undo()
+{
+	exit_stack_pop
+}
+exit_stack_push unset -f ramen_branch_checkout_undo
+ramen_image_name=ramen-operator:v0.Npr58
+exit_stack_push unset -v ramen_image_name
+ramen_build()
+{
+	ramen_branch_checkout ${1}
+	make -C ${1} docker-build IMG=${ramen_image_name} DOCKER_HOST=${DOCKER_HOST}
+	git --git-dir ${1}/.git --work-tree ${1} status -s
+	git --git-dir ${1}/.git --work-tree ${1} checkout -- config
+	ramen_branch_checkout_undo
+}
+exit_stack_push unset -f ramen_build
+kube_context_set()
+{
+	exit_stack_push kubectl config use-context $(kubectl config current-context)
+	kubectl config use-context ${1}
+}
+exit_stack_push unset -f kube_context_set
+kube_context_set_undo()
+{
+	exit_stack_pop
+}
+exit_stack_push unset -f kube_context_set_undo
+ramen_deploy()
+{
+	DOCKER_HOST=${DOCKER_HOST} minikube -p ${2} image load ${ramen_image_name}
```

Comment:
This command will not work on a podman based system (which most of Fedora is at present). An alternate to load images from podman to minikune is [here](https://github.com/ShyamsundarR/volrep-shim-operator/blob/main/hack/pushpodmantodocker_minikube.sh)

Reply:
Issue https://github.com/RamenDR/ramen/issues/76 tracks this.

Reply:
Podman support is now included in this pull request.  Thanks @ShyamsundarR for providing a way to use `minikube image load` with `podman`; namely by first doing a `docker save`, assuming `docker` is a `podman` alias.  It was tested on an Ubuntu 18.04 system.  I would appreciate it if someone would try it on a Fedora or RHEL8 system that does not support docker.